Upstream-status: https://lists.gnu.org/archive/html/jami/2024-03/msg00008.html

This makes it possible to use the system-provided qwindowkit library.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6d2dccfb..8dedff50 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -100,13 +100,13 @@ endif()
 
 # qwindowkit (frameless window)
 add_fetch_content(
-  TARGET qwindowkit
+  TARGET QWindowKit
   URL https://github.com/stdware/qwindowkit.git
   BRANCH 79b1f3110754f9c21af2d7dacbd07b1a9dbaf6ef
   PATCHES ${QWINDOWKIT_PATCHES}
   OPTIONS ${QWINDOWKIT_OPTIONS}
 )
-list(APPEND CLIENT_INCLUDE_DIRS ${QWindowKit_BINARY_DIR}/include)
+
 list(APPEND CLIENT_LIBS QWindowKit::Quick)
 
 set(CMAKE_AUTOMOC ON)
diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
index b2730b71..4960899e 100644
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -48,7 +48,7 @@ target_include_directories(test_common_obj PRIVATE
     ${CMAKE_CURRENT_SOURCE_DIR}
     ${CMAKE_SOURCE_DIR}/src)
 target_link_directories(test_common_obj PRIVATE ${CLIENT_LINK_DIRS})
-target_link_libraries(test_common_obj ${QML_TEST_LIBS})
+target_link_libraries(test_common_obj ${QML_TEST_LIBS} ${CLIENT_LIBS})
 target_compile_definitions(test_common_obj PRIVATE BUILD_TESTING="ON")
 
 set(COMMON_TESTS_SOURCES
